recipes/aquila/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:  
+  name: aquila
+  version: "1.0.0"
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+source:
+  url: https://github.com/xinzhoucs/Aquila/archive/v1.0.0.tar.gz
+  sha256: 035da84ffc116ce6a9cf5bdd2d53428efa2d84fd5c9009c94cc9c700410cc542
+
+requirements:
+  host:
+    - python >=3
+    - setuptools >=1.1
+
+  run:
+    - python >=3
+    - samtools
+    - pysam
+    - minimap2
+    - scipy
+    - numpy
+
+test:
+  commands:
+    - Aquila_step1 --help
+    - Aquila_step2 --help
+    - Aquila_clean --help
+    - Aquila_step1_multilibs --help
+    - Aquila_assembly_based_variants_call --help
+    - Aquila_phasing_all_variants --help
+    - Aquila_step0_sortbam --help
+    - Aquila_step0_sortbam_multilibs --help
+
+about:
+  home: https://github.com/maiziex/Aquila
+  license: MIT
```

Comment:
please include `license_file:`

And if you have please consider to include a DOI in the extra section.